config/config/sc-config.yaml

```diff
@@ -166,24 +166,24 @@ harbor:
         cpu: 2m
         memory: 15Mi
       limits:
-        cpu: 5m
-        memory: 20Mi
+        cpu: 100m
+        memory: 250Mi
   notarySigner:
     resources:
       requests:
         cpu: 2m
         memory: 10Mi
       limits:
-        cpu: 5m
-        memory: 20Mi
+        cpu: 100m
+        memory: 250Mi
```

Comment:
Some of these are quite big changes in limits (this is just one of them). Should we also increase the requests a bit?

Reply:
It is probably a good idea to increase the requests as well. To ensure that the pods can run when some load are put on them. Unsure what the best practices here are, should requests and limits always follow each other?

Reply:
I'd say that we don't have any strict requirements here. We don't always set request=limit, so you don't have to either.
You can use your judgement to set them to something sensible, if you have any data from when you wanted to increase the limits, then maybe base it on that.

Reply:
personally I set the limit = 2 x request